src/util/events.js

```diff
@@ -49,12 +56,6 @@ export function getFilteredEvents(events, filters, zipcodes) {
     }
 
     if (filters.zipcode) {
-      // We do not yet have valid zipcodes to filter against,
-      // give it a pass.
-      if (!zipcodes || !zipCodesLength) {
-        return true;
-      }
-
       const zipcodeLatLng = L.latLng(zipcodes[filters.zipcode]);
```

Comment:
Ah I guess that wouldn't happen with `validZipcode` check above, nevermind. (Was checking against the staging deployment.)